### PR TITLE
Pasteing of comma and space separated lists

### PR DESCRIPTION
**Pull request by bmordan**
This PR enables uses to paste a list of either:
`'comma','separates','values'`
or
`'words' 'separated' 'with' spaces`
and
have **react-tag-input** split the values up and add them one by one to the field.

Tidy 📋

### CODE REVIEW

**Comment by alanshaw** [comment]
👓

File: dist-modules/reactTags.js [hunk 1]
```diff
@@ -179,6 +178,20 @@ var ReactTags = React.createClass({
...
+        var _this = this;
+
+        e.preventDefault();
+        var string = e.clipboardData.getData('Text');
+        var commas = string.split(',');
+        var spaces = string.split(' ');
+
+        if (commas.length > spaces.length) commas.forEach(function (buzzword) {
+            return _this.props.handleAddition(buzzword);
+        });else spaces.forEach(function (buzzword) {
```
Comment (alanshaw) [inline dist-modules/reactTags.js]: Syntax error here

File: dist-modules/reactTags.js [hunk 2]
```diff
@@ -179,6 +178,20 @@ var ReactTags = React.createClass({
             });
         }
     },
+    handlePaste: function handlePaste(e) {
+        var _this = this;
+
+        e.preventDefault();
+        var string = e.clipboardData.getData('Text');
+        var commas = string.split(',');
+        var spaces = string.split(' ');
```
Comment (alanshaw) [inline dist-modules/reactTags.js]: Split on space or comma?

``` js
string.split(/[ ,]+/)
```

``` js
$ node
> 'and then, there,were,  some ,   words'.split(/[ ,]+/)
[ 'and', 'then', 'there', 'were', 'some', 'words' ]
```

**Comment by alanshaw** [comment]
You didn't make all the changes you made in `lib/reactTags.js` as you made in `dist-modules/reactTags.js`. I think the latter is generated anyway so you should probably use the build process for doing that if there is one.

File: lib/reactTags.js [hunk 3]
```diff
@@ -168,6 +167,11 @@ var ReactTags = React.createClass({
             });
         }
     },
+    handlePaste: function (e) {
+      e.preventDefault()
+      let string = e.clipboardData.getData('Text')
```
Comment (alanshaw) [inline lib/reactTags.js]: In IE11 this causes an error. We need to check for `e.clipboardData` and if it doesn't exist get it from `window` (where IE11 keeps it).

``` js
const clipboardData = e.clipboardData || window.clipboardData
const string = clipboardData.getData('text')
```

Also note that lowercase "text" seems to be the standard.

**Comment by bmordan** [comment]
I have removed and `.gitignore` the `/dist` and `dist-modules` folders.
I added a `postinstall` step to generate these files.
There is no need to persist these in the repo, and they are a bit confusing when looking at the diffing on a PR

**Comment by alanshaw** [comment]
In future please send unrelated changes as a separate PR

**Pull request merged**